store/src/java/com/zimbra/cs/filter/jsieve/AddHeader.java

```diff
@@ -154,6 +155,12 @@ protected void validateArguments(Arguments arguments, SieveContext context)
         }
 
         if (!StringUtil.isNullOrEmpty(headerName)) {
+            String tempHeaderName = headerName;
```

Comment:
should move this code to static method in FilterUtil class and call that method in AddHeader and EditHeaderExtension.